Added support for the ItemsToProcessDirectory option to read from the Azure

Grunt101 opened the pull request:
I have added support for the ItemsToProcessDirectory option to read from the Azure Portal Settings via CloudConfigurationManager. The reason for this is that by default the WebJobs target *D:\home\site\wwwroot\* However, I have just fired up a Ghost Blog and in that folder there are all the Node Modules etc, which aren't required to be Minified (The same goes for the images). E.g.

![ghost blog](https://cloud.githubusercontent.com/assets/6916342/6916578/c05bf3d0-d79d-11e4-9ed8-7a0bdb500378.png)

The pull request allows the user to specify the following *AZURE_MINIFIER_PATH* and *AZURE_IMAGE_OPTIMIZER_PATH*  in the Azure App Settings and then that folder would be used. If no App Setting is specified, then the previous default would be used.

madskristensen commented:
That's a great idea. I recently had the same issue and had to disable the SiteExtension/WebJob because of it. Can you share a screenshot on what your Azure configuration looks like from the portal?

sayedihashimi commented:
For websites/webjobs you can read the azure settings from env vars. I thought CloudConfigMgr was mostly webrole.
I'd like to avoid the addl dependency if possible.

Grunt101 commented:
You guys are correct regarding the Environment Variables. I have updated the code to reflect this and removed the dependency.

Here is what my App Settings in the portal looks like

![azure settings](https://cloud.githubusercontent.com/assets/6916342/6949110/d19b0698-d8b2-11e4-9657-7ef30561091e.png)

Grunt101 commented:
Hey guys, I was wondering what you thought of the pull request and if you wanted me to make any additional changes. If so, just let me know :)

sayedihashimi commented:
@Grunt101 I'm planning on reviewing/trying it out later today. Could you add some text to the readme.md stating how to use this feature?

sayedihashimi commented:
@Grunt101 I have merged it and pushed to release. Once the [build completes on release branch](https://ci.appveyor.com/project/sayedihashimi/azurejobs/build/1.0.57) it should be available as version ```0.0.21-beta```.
